com/stencyl/models/GameModel.hx

```diff
@@ -65,11 +67,11 @@ class GameModel
 		actualWidth = Std.parseInt(xml.att.awidth);
 		actualHeight = Std.parseInt(xml.att.aheight);
 		scale = Std.parseInt(xml.att.scale);
-		defaultSceneID = 0;
-		
+		defaultSceneID = IdUtils.INVALID_ID;
```

Comment:
Not equivalent, but this shouldn't be an issue because if the `defaultSceneID` can't be read, something is really wrong anyway.